tests/test_pipenv.py

```diff
@@ -796,3 +796,21 @@ def test_local_zipfiles(self):
             dep = p.lockfile['default'][key]
 
             assert 'file' in dep or 'path' in dep
+
+
+    @pytest.mark.install
+    @pytest.mark.files
+    @pytest.mark.urls
+    def test_install_remote_requirments(self):
+        with PipenvInstance() as p:
+            # using a github hosted requirements.txt file
+            c = p.pipenv('install -r https://github.com/kennethreitz/pipenv/blob/3688148ac7cfecefb085c474b092c31d791952c1/tests/test_artifacts/requirements.txt')
```

Comment:
This is going to need to be `https://raw.githubusercontent.com/kennethreitz/pipenv/3688148ac7cfecefb085c474b092c31d791952c1/tests/test_artifacts/requirements.txt` actually. The link I originally provided is a full HTML page from github, not the raw file.